### PR TITLE
feat: Database model for meetings

### LINKED ISSUE #8: [Feat]: Create database model for upcoming meetings
### My suggestion

Create db model with fields like:
- date of the metting
- title
- description

### Possible alternatives

_No response_

### PR DESCRIPTION
**Pull request by tapczanek1321**
Created database model that will hopefully work

### CODE REVIEW

**Review by Sanielia — APPROVED** [review]
LGTM uwu
Tho in future add (#issueID) to commits and PRs names

File: core/models.py [hunk 1]
```diff
@@ -1 +1,11 @@
+from django.db import models
 # Create your models here.
+
+
+class NextMeeting(models.Model):
+    title = models.CharField(max_length=50)
```
Comment (Sanielia) [inline core/models.py]: Okey, I didn't see this when I was approving. I think you should increase max-length to at least 150. 50 for title is certainly too litt;e

**Review by Sanielia — CHANGES_REQUESTED** [review]

File: core/models.py [hunk 2]
```diff
@@ -1 +1,11 @@
...
+    description = models.TextField()
+    date_of_meeting = models.DateTimeField("Data spotkania")
```
Comment (goteusz-maszyk) [inline core/models.py]: why did you put "Data spotkania" and not "Opis"?

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

File: core/models.py (hunk 2 above)
Comment (tapczanek1321) [reply core/models.py]: because date of meeting doesn't need a description (i think so)

File: core/models.py [hunk 3]
```diff
@@ -1 +1,11 @@
...
 # Create your models here.
+
+
+class NextMeeting(models.Model):
+    title = models.CharField(max_length=150)
+    description = models.TextField()
+    date_of_meeting = models.DateTimeField("Data spotkania")
+
+    def __str__(self):
+        return self.title
```
Comment (Norbiros) [inline core/models.py]: Add a bit more info here...

**Review by Norbiros — APPROVED** [review]

**Review by Sanielia — APPROVED** [review]

**Review by goteusz-maszyk — APPROVED** [review]

**Pull request merged**